docs/pipelines/tasks/_shared/yaml/DownloadPackageV1.md

```diff
@@ -4,7 +4,7 @@
 - task: DownloadPackage@1
   inputs:
     #packageType: 'nuget' # Options: maven, npm, nuget, pypi, upack
-    feed: 
```

Comment:
I would say "organization-scoped feeds" instead of "regular feeds", which implies others are irregular ;) and I'd just say "project-scoped feeds" instead of public feeds as well.

Reply:
Fixed